### ISSUE TITLE
Vulnerability notification

### ISSUE THREAD

**Issue by yamerooo123** [unopim/unopim#41]
### Terms

- [X] Before you open this issue, I have checked if the issue has already been reported.

### UnoPim Version(s) affected

0.1.2 and below

### Issue Description

Hello,

I am a security researcher, i perform security testing for open-source projects on GitHub for free.

This is a courtesy notification, I have identified a vulnerability in your product which affected v.0.1.2 and below.

Please review the issue through Security tab to find mitigations together. 

Thank you

### Preconditions

Please review the issue in Security tab.

### Steps to reproduce

Please review the issue in Security tab.

### Expected Result

Please review the issue in Security tab.

### Actual Result

Please review the issue in Security tab.

**Comment by devansh-pal-webkul** [comment]
Hi @yamerooo123 ,

Thank you for bringing this to our attention! We appreciate your effort in identifying potential security vulnerabilities.

We require some additional details for this please follow the below steps to report the vulnerability: 
* Go to the Security tab of the repository.
* Click on the Report a vulnerability button.
* Provide detailed information about the vulnerability, including:
  - Steps you followed to reproduce the issue
  - Any relevant context
Your input is invaluable to us in maintaining the security of our project.

Thanks again for your support!

**Comment by yamerooo123** [comment]
Yes. I have provided additional details. Please check the Security tab. Thank you!

**Comment by devansh-pal-webkul** [comment]
Thank you for reporting this security issue. We are actively working to resolve this issue and will try to add it in the next release or as a patch once resolved.

**Comment by navneetbhardwaj** [comment]
Fixed in

**Comment by yamerooo123** [comment]
Hello,

Since the vulnerability has been patched, I would like to proceed with requesting a CVE ID.
A CVE will raise user awareness to upgrade the application to the latest version. This ensures user security and demonstrates that your application prioritizes user safety. I will handle everything in the process and notify the vendors again once the CVE is released. In the future, if I discover more vulnerabilities, I will ensure that the process is done discreetly again.

Thank you.

**Comment by navneetbhardwaj** [comment]
@yamerooo123 

Thank you for notifying us about the assignment and public disclosure of CVE-2024-50637. We appreciate your prompt communication on this matter.
